client/src/components/CardBackgroundImage/CardBackgroundImage.stories.js

```diff
@@ -0,0 +1,29 @@
+import _ from "lodash";
+import React from "react";
+
+import { trivial_text_maker } from "src/models/text.js";
+
+import { CardBackgroundImage } from "./CardBackgroundImage.js";
+
+export default {
+  title: "CardBackgroundImage",
+  component: CardBackgroundImage,
+};
+
+// not sure if the component works or not
+const Template = (args) => <CardBackgroundImage {...args} />;
```

Comment:
I believe it works, although it's kind of a legacy component and no longer used. To look right, I think it needed to inherit a background colour from the consumer/and or uses the image as the background. Not going to look right, or be able to see the text, if an image src isn't supplied. You can test it by grabbing any image url from anywhere on the internet (just for now, we should use static images we host ourselves for things like this).

IMO, with an unused component like this one, where the proper usage isn't clear enough to storybook easily, I think it's worth considering deleting it. Maybe flag components like this in your PRs or bring them up on Slack. If no one can think of a reason to keep them we can toss them out. If they seem useful but are otherwise weird then we should open an issue to clean them up.

Reply:
Followed the suggestions that you gave and implemented them!